Data/market-hours/market-hours-database.json

```diff
@@ -4190,7 +4252,9 @@
       ],
       "saturday": [],
       "holidays": [
-        "4/14/2017"
+        "4/14/2017",
+        "12/25/2017",
+        "1/1/2018"
       ]
```

Comment:
A few more holidays for the Italian exchange (2017): 
4/17/2017, 5/1/2017, 8/15/2017, 12/26/2017

Official trading calendar: 
http://www.borsaitaliana.it/borsaitaliana/calendario-e-orari-di-negoziazione/calendario2017_pdf.htm